pkg/node/address.go

```diff
@@ -233,6 +233,14 @@ func SetInternalIPv4Router(ip net.IP) {
 	})
 }
 
+// GetInternalIPv6Router returns the cilium internal IPv6 node address. This must not be conflated with
+// k8s internal IP as this IP address is only relevant within the Cilium-managed network (this means
+// within the node for direct routing mode and on the overlay for tunnel mode).
+func GetInternalIPv6Router() net.IP {
+	n := getLocalNode()
+	return n.GetCiliumInternalIP(true)
+}
+
```

Comment:
drive-by - note that we already have `GetIPv6Router()`. Although this addition doesn't even seem to be used ...?

Reply:
Sorry, I missed submitting a check logic for router ipv6 address.

Please take a look again, thanks.